builtin/common/after.lua

```diff
@@ -21,8 +21,8 @@ core.register_globalstep(function(dtime)
 			job.func(unpack(job.arg))
 			jobs[i] = jobs[#jobs]
 			jobs[#jobs] = nil
-		else
-			time_next = math.min(time_next, job.expire)
+		elseif job.expire < time_next then
+			time_next = job.expire
```

Comment:
`math.min` is faster than the extra branch. Please revert this change.

Reply:
Sorry, never mind. This code won't be compiled, and in interpreted mode the `if` version is faster.